tests/unit/test_tableclasses_cql_generation.py

```diff
@@ -279,8 +282,9 @@ def test_multi_clustering_column_metadata_vector_cassandra_table(
         mock_db_session.assert_last_equal(
             [
                 (
-                    "INSERT INTO k.tn (metadata_s, row_id_0, row_id_1, partition_id) VALUES (?, ?, ?, ?);",  # noqa: E501
+                    "INSERT INTO k.tn (attributes_blob, metadata_s, row_id_0, row_id_1, partition_id) VALUES (?, ?, ?, ?, ?);",  # noqa: E501
                     (
+                        None,
```

Comment:
it seems we don't have a single CQL generation test where `attributes_blob` is being set to a value other than None.

Maybe you could add one instance of this?

Reply:
Good call - added a whole test function around the metadata splitting in the (now different) read and write paths.